### ISSUE TITLE
Date Formula returns null.

### ISSUE THREAD

**Issue by Scantheus** [notion-dotnet/notion-sdk-net#123]
A formula like dataAdd(prop("StartDate"), 5, "weeks") does not get result.
Notion.Client.DatePropertyValue.Date is null.

**Comment by KoditkarVedant** [comment]
@Scantheus can you provide more details like: `package version`, `api endpoint` you were using.

If you were interested you can also open a PR to fix the issue, I will be happy to merge it.

**Comment by Scantheus** [comment]
v1.4 (latest)
NotionClient.Databases.QueryAsync
NotionClient.Pages.RetrieveAsync

To make sure it wasn't the Notion API itself, I checked it in an app using RestSharp and it works fine.
I haven't checked bool or string formulas but a number formula works fine.

Sorry, finding a solution is beyond my capabilities but I've tracked it down to ParseStreamAsync.
In debugging, I looked at response returned by SendAsync in GetAsync in RestClient.cs.
The date in the date formula property is fine in response.Content but ParseStreamAsync isn't pulling it out.
